src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_params.py

```diff
@@ -173,6 +173,7 @@ def load_arguments(self, _):
         c.argument('service_cidr')
         c.argument('vnet_subnet_id')
         c.argument('workspace_resource_id')
+        c.argument('skip_role_assignment', required=False, action='store_true')
```

Comment:
what does `action='store_true'` mean?

Reply:
when specified, the value for the flag is true.
```                    details['options']['action'] = 'store_true'
                    self._help(name, "True if flag present.")
```